tests/functional/docs/test_help_output.py

```diff
@@ -177,6 +177,10 @@ def test_elb_help_output(self):
         # .. _`:
         self.assert_not_contains('.. _`:')
 
+    def test_shorthand_flattens_list_of_single_member_structures(self):
```

Comment:
I think this comment still stands. Could we add functional tests (one for each service) to make sure the backcompat works.